vimeo/VimeoUpload
# Fixing missing VimeoNetworking dependency

#### Ticket
**Required for Vimeans only**
N/A

#### Ticket Summary
Cameo is failing to build VimeoUpload because CocoaPods is not creating the correct target dependencies for VimeoUpload.  The VimeoUpload pod spec file removed the dependency on VimeoNetworking which is causing the issue.

#### Implementation Summary
I added back the s.dependency 'VimeoNetworking' line, which instructs Cocoapods to create the target dependencies for VimeoUpload on VimeoNetworking and AFNetworking.  I removed the version number pinning because we specify a git sha in the pod file.

#### How to Test
1. Run pod install
2. Make sure all the example projects compile.
